packages/fx-core/src/plugins/resource/bot/v2/common.ts

```diff
@@ -42,7 +42,10 @@ export function decideTemplateScenarios(ctx: Context, inputs: Inputs): Set<strin
         templateScenarios.add(TemplateProjectsScenarios.COMMAND_AND_RESPONSE_SCENARIO_NAME);
         break;
       case BotScenario.NotificationBot:
-        const notificationTriggerType = inputs[QuestionNames.BOT_HOST_TYPE_TRIGGER] as string[];
```

Comment:
`inputs` may not contain trigger type.

Reply:
Is this case in ut? Notification bot should always contain trigger type.

Reply:
Good catch for defensive code

Reply:
No. I think this is a bug of question model. 
`teamsfx add notification --interactive false` add a notification without any trigger.